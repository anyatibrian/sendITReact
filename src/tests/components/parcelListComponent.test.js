import React from "react";
import {shallow} from "enzyme";
import ParcelListComponent from "../../components/parcelListComponent";

let wrapper;
let props = {
  parcelOrder:{map:jest.fn()}
};
describe('parcel list component', ()=>{
  beforeEach(()=>{
    wrapper = shallow(<ParcelListComponent {...props}/>)
  });
  it('should render without fail', ()=> {
    expect(wrapper).toMatchSnapshot();
  });
});