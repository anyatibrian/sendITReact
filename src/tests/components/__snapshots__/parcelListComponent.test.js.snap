// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`parcel list component should render without fail 1`] = `ShallowWrapper {}`;
